### PR TITLE
Update Sidecar documentation to 1.0

### PR DESCRIPTION
**Pull request by mpfz0r**
- Note about Sidecar vs. Collectors (legacy)
- Describe new sidecar.yml
- Update step-by-step guide

### CODE REVIEW

File: pages/sidecar.rst [hunk 1]
```diff
@@ -1,31 +1,33 @@
-.. _graylog-collector-sidecar:
+.. _graylog-sidecar:
```
Comment (jalogisch) [inline pages/sidecar.rst:1]: Did you checked the documentation if this old :Ref: is used somewhere?

File: index.rst [hunk 2]
```diff
@@ -21,7 +21,7 @@ NOTE: There are multiple options for reading this documentation. See link to the
    pages/configuration
    pages/secure/securing
    pages/sending_data
-   pages/collector_sidecar
```
Comment (jalogisch) [inline index.rst:24]: we should include a `collector_sidecar` page with the `:orphan:` header that gives notice that this is no longer present and a link to the new page.

File: pages/sidecar.rst [hunk 3]
```diff
@@ -37,64 +39,91 @@ To get the Sidecar working `Download a package <https://github.com/Graylog2/coll
...
+    $ vi /etc/graylog/sidecar/sidecar.yml
 
-    $ sudo graylog-collector-sidecar -service install
+    $ sudo graylog-sidecar -service install
 
     [Ubuntu 14.04 with Upstart]
-    $ sudo start collector-sidecar
+    $ sudo start graylog-sidecar
 
     [Ubuntu 16.04 with Systemd]
```
Comment (jalogisch) [inline pages/sidecar.rst]: maybe ad `Ubuntu 16.04  and later` ? as 16.04 is EOL in April 2019

File: pages/sidecar.rst [hunk 4]
```diff
@@ -148,170 +159,98 @@ system service. We just need the binaries installed on the system::
...
+| Parameter                           | Description                                                                                                         |
++=====================================+=====================================================================================================================+
+| server_url                          | URL to the Graylog API, e.g. ``http://192.168.1.1:9000/api/``                                                       |
++-------------------------------------+---------------------------------------------------------------------------------------------------------------------+
+| server_api_token                    | The API token to use to authenticate against the Graylog server API. |br|                                           |
+|                                     | e.g ``1jq26cssvc6rj4qac4bt9oeeh0p4vt5u5kal9jocl1g9mdi4og3n``  |br|                                                  |
+|                                     | The token is not optional and needs to be configured.                                                               |
```
Comment (jalogisch) [inline pages/sidecar.rst]: > not optional 

Maybe `mandatory` is the better choice? 

People will not notice the `not` and read optional...

**Review by jalogisch — CHANGES_REQUESTED** [review]
Some rewordings and thoughts

**Review by jalogisch — APPROVED** [review]

**Review by mariussturm — APPROVED** [review]

**Pull request merged**